File: GarridoJulian_GenerarTiempos.py
import numpy as np
import matplotlib.pyplot as plt
import time
N = 35
a =[]
b = []
i = 0
def fibonacci(N):
	if (N==0 or N==1):
		return N
	else:
		return fibonacci(N-2)+fibonacci(N-1)


def get_time(N):
	t0 = time.time()
	fibonacci(N)
	t1 = time.time()-t0
	return t1


for i in range(N):
	
	print (i,get_time(i))

File: GarridoJulian_GenerarTiempos.cpp
#include <iostream>
#include <ctime>
using namespace std;
// Se crea la serie de fibonacci

int fibonacci( int N){

 if(N == 0 || N == 1){
       return N;
  }
  else{
return fibonacci(N - 2) + fibonacci(N - 1);
}
}
// se cre un metodo que cuenta el tiempo que tarda enh ejecutarse el metodo de la serie de fibonacci
int get_time(int N){
 clock_t t;
t = clock();
fibonacci(N);
t = clock() - t;
float time = ((float)t)/CLOCKS_PER_SEC;
return time;

}

int main(){
int n = 35 ;
int i = 0;


while(i<n){
 cout<< get_time(i);
i++;
}

return 0;
}


